runtime/caml/dynlink.h

```diff
@@ -41,11 +41,10 @@ extern void caml_build_primitive_table_builtin(void);
 /* Unload all the previously loaded shared libraries */
 extern void caml_free_shared_libs(void);
 
-/* Return the effective location of the standard library */
-extern const char_os * caml_get_stdlib_location(void);
-
-/* Parse ld.conf and add the lines read to caml_shared_libs_path */
-extern char_os * caml_parse_ld_conf(void);
+/* If found, parse $OCAMLLIB/ld.conf, $CAMLLIB/ld.conf and stdlib/ld.conf in
+   that order and add the lines read to table. */
+extern char_os * caml_parse_ld_conf(const char_os * stdlib,
```

Comment:
FYI: I did a GitHub search for `caml_parse_ld_conf` since adding arguments to the argument list of `caml_parse_ld_conf` is veering into C undefined behavior (if client code accesses an old `caml_parse_ld_conf` in shared library, or worse if vice versa). No one outside of startup_byt.c is using it, so it looks safe to change.